demo/src/features/debug3d/extract.rs

```diff
@@ -1,28 +1,25 @@
-use crate::features::debug3d::plugin::Debug3DStaticResources;
-use crate::features::debug3d::prepare::Debug3dPrepareJobImpl;
-use crate::features::debug3d::{Debug3dRenderFeature, DebugDraw3DResource, ExtractedDebug3dData};
+rafx::declare_render_feature_extract_job!();
```

Comment:
I don't want to use macros to add use statements. An alternative would be to do something like:

```
pub mod job_prelude {
    pub use crate::features::etc;
}
```

and then use crate::job_prelude::*;